### ISSUE TITLE
Fix failing integration tests

### ISSUE THREAD

**Issue by markdboyd** [cloud-gov/opensearch-dashboards-cf-auth-proxy#66]
## Background

Tests are currently failing in CI with these errors:

```shell
FAILED e2e/test_discover_filters_logs.py::test_see_correct_logs_in_discover_tag[firefox] - playwright._impl._api_types.TimeoutError: Timeout 30000ms exceeded.
FAILED e2e/test_discover_filters_logs.py::test_see_correct_logs_in_discover_tag_2[firefox] - playwright._impl._api_types.TimeoutError: Timeout 30000ms exceeded.
FAILED e2e/test_setup.py::test_login_redirects_home_without_slash[firefox] - playwright._impl._api_types.TimeoutError: Timeout 30000ms exceeded.
FAILED e2e/test_setup.py::test_login_redirects_home[firefox] - playwright._impl._api_types.TimeoutError: Timeout 30000ms exceeded.
FAILED e2e/test_setup.py::test_login_remembers_target[firefox] - playwright._impl._api_types.TimeoutError: Timeout 30000ms exceeded.
```

## Acceptance Criteria

- [x] e2e tests pass in CI

**Comment by markdboyd** [comment]
Fixed via #67

**Comment by markdboyd** [comment]
The last three runs of the e2e tests have passed with the latest changes:

https://ci.fr.cloud.gov/teams/main/pipelines/opensearch-dashboards-cf-auth-proxy/jobs/e2e/builds/97
https://ci.fr.cloud.gov/teams/main/pipelines/opensearch-dashboards-cf-auth-proxy/jobs/e2e/builds/98
https://ci.fr.cloud.gov/teams/main/pipelines/opensearch-dashboards-cf-auth-proxy/jobs/e2e/builds/99
